doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+   +=============+=============+=============+=============+
+   |ARM7-32bit   |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+   |i386         |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+
+Community
+~~~~~~~~~
+
+.. table::
+   :widths: 15 15 30 40
+   :width: 100%
+
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |Architecture |Support      |QA                         |Notes                                        |
+   +=============+=============+===========================+=============================================+
+   |PPC64el      |             |Part of Fedora automated QA|Access can be arranged through IBM dev cloud |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |PPC64        |Victor Julien|                           |No access except an old Mac G5 Victor has    |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |PPC32        |Victor Julien|                           |No access except an old Mac G4 Victor has    |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+
+High Level Features
+-------------------
+
+Capture support
+~~~~~~~~~~~~~~~
+
+Tier 1
+^^^^^^
+
+.. table::
+   :width: 100%
+
+   +----------------+-------------------------+----+-----------------------------+
+   | Capture Type   | Maintainer              | QA | Notes                       |
+   +================+=========================+====+=============================+
+   |AF_PACKET       |Core team / Eric Leblond |    |Used by Security Onion, SELKS|
+   +----------------+-------------------------+----+-----------------------------+
+   |NETMAP (FreeBSD)|Core team / Victor Julien|    |Used by OPNsense             |
+   |                |+ Aleksey Katargin       |    |                             |
+   +----------------+-------------------------+----+-----------------------------+
+   |NFQUEUE         |Core team / Andreas Herz |    |                             |
+   +----------------+-------------------------+----+-----------------------------+
+   |libpcap         |Core team                |    |                             |
+   +----------------+-------------------------+----+-----------------------------+
+
+Tier 2
+^^^^^^
+
+.. table::
+   :width: 100%
+
+   +--------------------+-------------------------+----+---------------+
+   |Capture Type        |Maintainer               |QA  |Notes          |
+   +====================+=========================+====+===============+
+   |PF_RING             |Core team                |    |               |
+   +--------------------+-------------------------+----+---------------+
+   |AF_PACKET (eBPF/XDP)|Eric Leblond             |    |Bleeding edge  |
```

Comment:
should this move to community support?

Reply:
With more and more people trying to use it, it might be worth to make it obvious that it's not 100% supported by us or we will have to invest more to bump it to that kind of Tier.

Reply:
Moved to community.